dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:8-alpine AS build
+
+# Create the app directory
+WORKDIR /app
+
+# Copy package json
+COPY package*.json ./
+
+FROM build as publish
+RUN npm build
+
+FROM node:8-alpine AS runtime
+WORKDIR /app
+
+COPY --from=publish /app/package*.json ./
```

Comment:
Copy from dist is also required, since the docker app wont start otherwise. Even though it will build successfully.